docker/Dockerfile.ci_adreno

```diff
@@ -22,10 +22,7 @@ COPY utils/apt-install-and-clear.sh /usr/local/bin/apt-install-and-clear
 
 # Android SDK
 COPY install/ubuntu_install_androidsdk.sh /install/ubuntu_install_androidsdk.sh
-RUN bash /install/ubuntu_install_androidsdk.sh
-ENV ANDROID_HOME=/opt/android-sdk-linux
-ENV ANDROID_NDK_HOME=/opt/android-sdk-linux/ndk/21.3.6528147
-ENV ANDROID_NDK_MAJOR=21
+RUN bash /install/ubuntu_install_androidsdk.sh 25.2.9519653 25
```

Comment:
Sorry, one more comment. Probably we shouldn't pass `ANDROID_NDK_MAJOR`? We can parse this value from `ANDROID_NDK_VERSION`.

Reply:
Done. Made few other tools versions too configurable. Cmake was causing some warnings over older version with new NDK.